docs/components/utilities/package.md

```diff
@@ -1,4 +1,124 @@
 # Package
 
-!!! warning
-    This page is still in progress. You can check the description of blocks for help.
+## Overview
+**Package** is a non-visible component that allows users to get some information about the application packages. 
+
+## Properties
+
+### App Name
+
+![App Name](/assets/images/components/utilities/package/p_app-name.png)
+
+_**\(** Getter only **\)**_
+
+`Type: Number`
+
+Returns the name of the currrent running application.
+
+### Package Name
+
+![Package Name](/assets/images/components/utilities/package/p_package-name.png)
+
+_**\(** Getter only **\)**_
+
+`Type: Number`
+
+Returns the package name of the currrent running application.
+
+### Version Code
+
+![Version Code](/assets/images/components/utilities/package/p_version-code.png)
+
+_**\(** Getter only **\)**_
+
+`Type: Number`
+
+Returns the version code of the currrent running application.
+
+### Version Name
+
+![Version Name](/assets/images/components/utilities/package/p_version-name.png)
+
+_**\(** Getter only **\)**_
+
+`Type: Alphanumerical`
+
+Returns the version name of the currrent running application.
+
+## Methods
+
+### App Name From
+
+![App Name From](/assets/images/components/user-interface/utilities/package/m_app-name-from.png)
+
+_**\(** Text `package Name` **\)**_
+
+Returns the name of the app from the given package name.
+
+Params               |  []()       
+-------------------- | ------- 
+`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, `io.kodular.myfirstapp`
```

Comment:
Examples has to be with italics, not with code (as code is used for the param name)

```suggestion
`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, _io.kodular.myfirstapp_.
```